build.gradle

```diff
@@ -264,8 +264,8 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.13.5-prerelease'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.14.1-prerelease'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
```

Comment:
This needs to be updated to a full release once def file is merged https://github.com/hmcts/ccd-test-definitions/pull/41

Reply:
can this be updated now that the test-definitions PR has been merged?

Reply:
Updated